### PR TITLE
services: fix alias units not showing details and text filter

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pkg/systemd/services/services.jsx [hunk 1]
```diff
@@ -704,8 +704,8 @@ class ServicesPage extends React.Component {
                     if (unit.LoadState == "not-found")
                         return false;
 
-                    if (currentTextFilter && unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter) == -1 &&
-                        unit.Id.indexOf(currentTextFilter) == -1 && !(unit.Names || []).some(name => name.indexOf(currentTypeFilter) != -1))
+                    if (currentTextFilter && !((unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter.toLowerCase()) != -1) ||
+                        unit_id.indexOf(currentTextFilter) != -1))
```
Comment (marusak) [inline pkg/systemd/services/services.jsx:708]: Shouldn't then we always check case insensitive even for `unit_id`?

Comment (marusak) [inline pkg/systemd/services/services.jsx:708]: `unit_id` is not the same as `unit.Id`. For example:
`unit_id` for `ModemManager.service` (this is `unit.Id`) is `dus-org.freedesktop.ModemManager1.service`
Also you seem to drop `unit.Names` check ( agree with this, lets filter only by things we show in the UI and not keep some things magically on) but maybe just mention it in the commit messsage?
Plus can we get test for this? (mainly the case insensitive matching for both name and description).

File: pkg/systemd/services/services.jsx [hunk 2]
```diff
@@ -705,7 +705,7 @@ class ServicesPage extends React.Component {
                         return false;
 
                     if (currentTextFilter && unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter) == -1 &&
-                        unit.Id.indexOf(currentTextFilter) == -1)
+                        unit.Id.indexOf(currentTextFilter) == -1 && !(unit.Names || []).some(name => name.indexOf(currentTypeFilter) != -1))
```
Comment (marusak) [inline pkg/systemd/services/services.jsx]: so this commit introduced this and then next commit dropped it?

**Review by marusak — CHANGES_REQUESTED** [review]

File: pkg/systemd/services/services.jsx (hunk 2 above)
Comment (KKoukiou) [reply pkg/systemd/services/services.jsx]: Mistake - I did want that it.

File: pkg/systemd/services/services.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/systemd/services/services.jsx:708]: This is a deliberate decision. I wrote why in the commit message.

Comment (KKoukiou) [reply pkg/systemd/services/services.jsx:708]: Done.

**Review by marusak — DISMISSED** [review]
Thank you!

**Comment by martinpitt** [comment]
NameError: name 'wait_service_present' is not defined

**Comment by martinpitt** [comment]
Fails on [coreos](https://logs-https-cockpit.apps.ci.centos.org/logs/pull-14621-20200915-082014-08e690a1-fedora-coreos/log.html#121-1), with an [interesting error](https://logs-https-cockpit.apps.ci.centos.org/logs/pull-14621-20200915-082014-08e690a1-fedora-coreos/TestServices-testBasic-fedora-coreos-127.0.0.2-2202-FAIL.png).

**Review by martinpitt — APPROVED** [review]

**Pull request merged**